### ISSUE TITLE
ROS2 branches with Python

### ISSUE THREAD

**Issue by RodBelaFarin** [ros-perception/laser_geometry#79]
All ROS2 branches do still include the Python wrapper, which start with importing rospy - so the Python part is still ROS1. I find it quite confusing to have a ROS2 package with non-working ROS1 code inside.

**Comment by jonbinney** [comment]
Good point. I went ahead and updated the python code to use ros2: https://github.com/ros-perception/laser_geometry/pull/80

**Comment by ahcorde** [comment]
PR was merged
